src/app.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Inject, Injectable, OnApplicationBootstrap } from '@nestjs/common';
+import { ConfigType } from '@nestjs/config';
+import authConfig from './config/auth.config';
+import { AdminRepository } from './repositories/admin.repository';
+
+@Injectable()
+export class AppService implements OnApplicationBootstrap {
+  constructor(
+    @Inject(authConfig.KEY)
+    private readonly authConf: ConfigType<typeof authConfig>,
+    private readonly adminRepository: AdminRepository,
+  ) {}
+
+  async onApplicationBootstrap() {
+    const admin = await this.adminRepository.findByUsername(
+      this.authConf.adminUsername,
+    );
+    if (!admin && true) {
```

Comment:
```suggestion
    if (!admin && true) {
```
Why do we need that `true` value?

Reply:
this is something that shouldn't be there :see_no_evil: